src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -218,6 +196,93 @@
         }
     }
 
+    private ResponseEntity<?> getListedSubscriptions(String subscriptionName) {
+        // set is used to prevent subscription names repeating
+        Set<String> subscriptionNameList = new HashSet<>(Arrays.asList(subscriptionName.split(",")));
+        List<Subscription> foundSubscriptionList = new ArrayList<>();
+        List<String> notFoundSubscriptionList = new ArrayList<>();
+
+        subscriptionNameList.forEach(name -> {
+            try {
+                LOGGER.debug("Subscription fetching has been started: {}", name);
+                // Make sure the password is not sent outside this service.
+                Subscription subscription = subscriptionService.getSubscription(name);
+                subscription.setPassword("");
+                foundSubscriptionList.add(subscription);
+                LOGGER.debug("Subscription [{}] fetched successfully.", name);
+            } catch (SubscriptionNotFoundException e) {
+                LOGGER.error("Subscription not found: {}", name);
+                LOGGER.debug("Subscription not found traceback:\n {}", e);
```

Comment:
Same comment here to connect the two log messages, it's easier to have the subscription name to search for in the logs

Reply:
I just removed it. There might be others arguing it should stay though. We'll see.